### PR TITLE
Update azure_rm_sqlserver related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_sqlserver related document

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_sqlserver/azure_rm_sqlserver_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_sqlserver.py [hunk 1]
```diff
@@ -41,15 +41,14 @@
...
         description:
-            - "The identity type. Set this to 'SystemAssigned' in order to automatically create and assign an Azure Active Directory principal for the resour
-               ce. Possible values include: 'SystemAssigned'"
+            - The identity type. Set this to C(SystemAssigned) in order to automatically create and assign an Azure Active Directory principal for the resource.
+            - Possible values include C(SystemAssigned).
     state:
         description:
-            - Assert the state of the SQL server. Use C(present) to create or update a server and
-              C(absent) to delete a server.
+            - Assert the state of the SQL server. Use C(present) to create or update a server and use C(absent) to delete a server.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_sqlserver.py]:
```suggestion
            - State of the SQL server. Use C(present) to create or update a server and use C(absent) to delete a server.
```

**Review by acozine — COMMENTED** [review]
Thanks @Fred-sun - just one small suggested change to this one.

**Comment by Fred-sun** [comment]
@acozine Thanks for your review. Update new.

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by acozine** [comment]
Thanks @Fred-sun!

**Pull request merged**